tests/unit_tests/test_wave_06.py

```diff
@@ -119,7 +120,27 @@ def test_swap_best_by_category():
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
 
-@pytest.mark.skip
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3 
+
+    assert item_d.get_category() ==  "Clothing" and item_f.get_category() == "Clothing" and item_e.get_category() != "Clothing"
```

Comment:
Thorough is good generally, and maybe there might be a need to test that these items haven't changed categories, in this case that wasn't a possibility, so no need to test for it.